lib/mintel/config.libsonnet

```diff
@@ -1,6 +1,11 @@
 {
   _config+:: {
     local this = self,
+
+    // Set namespace here to allow jsonnet rendering from this directory.
+    // Note, this will get overwritten by the main config (default is 'monitoring)'...
+    namespace: 'monitoring',
```

Comment:
can we use a `fake namespace` ? like `unspecified` ? or do we need a real one ?

Reply:
Ideally you do need a real one, because if you are developing on a dashboard that uses the namespace (cost-one for example), you want the real namespace.